alibuild_helpers/args.py

```diff
@@ -89,14 +86,23 @@ def doParseArgs(star):
                             help=("Fetch updates to repositories in MIRRORDIR. Required but nonexistent "
                                   "repositories are always cloned, even if this option is not given."))
 
-  build_parser.add_argument("--no-local", dest="noDevel", metavar="PKGLIST", default="", type=csv_list,
+  build_parser.add_argument("--no-local", dest="noDevel", metavar="PACKAGE", default=[], action="append",
```

Comment:
Does this mean it will not support comma separated lists anymore? I do not think this is acceptable.

Reply:
No, 2ebab24515ab9087c4c3998e5e5cb94589ca46a7 fixes this so that `--no-devel` and `--force-rebuild` work like `--disable` (in that `--disable A,B` and `--disable A --disable B` are equivalent).